_disability-benefits/conditions/exposure-to-hazardous-materials/specific-environmental-hazards.md

```diff
@@ -1,69 +1,32 @@
 ---
```

Comment:
here too!
https://vets-website-pr-2410.herokuapp.com/disability-benefits/conditions/exposure-to-hazardous-materials/specific-environmental-hazards/